pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -6,6 +6,9 @@
 , abseil-cpp
 , cmake
 , eigen
+, flatbuffers
+, gbenchmark
+, glibcLocales
```

Comment:
need to support `#include <locale>` in `onnxruntime/core/providers/cpu/text/string_normalizer.cc` (& probably elsewhere).

without this a bunch of `ContribOpTest`s fail (`StringNormalizer*` & `WordConvEmbedding*`)